### ISSUE TITLE
Chapter01\GuiseppeToys wrong set of data

### ISSUE THREAD

**Issue by pkktao** [PacktPublishing/Deep-Learning-with-PyTorch-Quick-Start-Guide#3]
I think there's a mix-up in the training set given in the 'Chapter01\GuiseppeToys' folder.

number of images in the folder: 89
number of images in the labels.csv: 122
And the filenames in both sources don't match.

**Comment by ignacioruiz** [comment]
Actually, labels.csv lists EXACTLY the complementary set of images. Seems nobody did a thorough code review.
